ios-sdk.xcodeproj/project.pbxproj

```diff
@@ -237,9 +241,20 @@
 			path = fonts;
 			sourceTree = "<group>";
 		};
+		DB366BDC1DF8BE090045C15F /* Images */ = {
+			isa = PBXGroup;
+			children = (
+				DB366BDD1DF8BE090045C15F /* ic_find_me_normal@2x.png */,
```

Comment:
They don't show the button. I actually realized this after I submitted it, but noticed I didn't have a large enough button asset for it, so @souperneon hooked me up - https://github.com/mapzen/ios/issues/104

I'll be updating this PR in a few with the new assets.